src/js/api.js

```diff
@@ -2,8 +2,24 @@ import fetch from 'isomorphic-fetch'
 
 let accessToken
 
+function createQueryString(suffix, params, length) {
```

Comment:
I'd prefer this as 

```
function createQueryString(suffix, params) {
  const length = Object.keys(params).length
  let newSuffix = `${suffix}?`
  let count = 0
  for(let key of Object.keys(params)) {
    newSuffix = `${newSuffix}${key}=${params[key]}`
    count++
    if(count < length) newSuffix = `${newSuffix}&`
  }
  return newSuffix
}
```

Since the length will always be calculated the same way